File: Codesignal/Eric/70.AlphabetSubsequence.go
package main 

import "fmt"

func solution(s string) bool {
	for i:=0; i<len(s)-1; i++ {
		if s[i] >= s[i+1] {
			return false;
		}
	}
	return true;
}

func main() {
	fmt.Println(solution("effg"));
}

File: Codesignal/Eric/70.AlphabetSubsequence.js
// function solution(s) {
//     for (let i=0; i<s.length-1; i++) {
//         if (s[i] >= s[i+1]) return false;
//     }
//     return true;
// }

function solution2(s) {
    return [...new Set(s)].sort().join('') === s;
}

console.log(solution2('effg')); // false